File: CS/MyBarManager/MyBarAndDockingController.cs
using System;
using DevExpress.XtraBars;
using System.ComponentModel;

namespace WindowsApplication1 {
    public class MyBarAndDockingController : BarAndDockingController {
        public MyBarAndDockingController(IContainer container) : base() { }
        public MyBarAndDockingController() : base() { }
        protected override void RegisterPaintStyles() {
            base.RegisterPaintStyles();
            PaintStyles.Add(new MySkinBarManagerPaintStyle(PaintStyles));
        }
    }
}

File: VB/MyBarManager/MyBarAndDockingController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraBars
Imports System.ComponentModel

Namespace WindowsApplication1
	Public Class MyBarAndDockingController
		Inherits BarAndDockingController
		Public Sub New(ByVal container As IContainer)
			MyBase.New()
		End Sub
		Public Sub New()
			MyBase.New()
		End Sub
		Protected Overrides Sub RegisterPaintStyles()
			MyBase.RegisterPaintStyles()
			PaintStyles.Add(New MySkinBarManagerPaintStyle(PaintStyles))
		End Sub
	End Class
End Namespace
